tutorial/practicals/LFRic/building_code/2_built_ins/Makefile

```diff
@@ -68,38 +68,45 @@ EXAMPLE_OBJ = $(DRIVER_OBJ) $(ALGORITHM_OBJ) $(PSY_OBJ)
 # Targets
 .DEFAULT_GOAL := compile
 
-.PHONY: compile transform clean
+.PHONY: transform compile run clean allclean
+
+# Test PSyclone target for continuous integration
+transform: $(PSY_SRC) $(ALGORITHM_SRC)
 
 compile: transform $(EXEC)
 
+run: compile
+	$(EXEC)
+
 $(EXEC): $(LFRIC_LIB) $(EXAMPLE_OBJ)
-	$(F90) $(F90FLAGS) $(EXAMPLE_OBJ) -o $(EXEC) -L$(LFRIC_PATH) -l$(LFRIC_NAME)
+	$(F90) $(F90FLAGS) $(LFRIC_INCLUDE_FLAGS) $(EXAMPLE_OBJ) \
+	-o $(EXEC) -L$(LFRIC_PATH) -l$(LFRIC_NAME)
 
 $(LFRIC_LIB):
 	$(MAKE) -C $(LFRIC_PATH)
 
 # Dependencies
-$(DRIVER_OBJ): $(ALGORITHM_OBJ) $(PSY_OBJ)
-$(ALGORITHM_OBJ): $(PSY_OBJ)
+$(DRIVER_OBJ): $(ALGORITHM_OBJ) $(PSY_OBJ) $(LFRIC_LIB)
```

Comment:
No need for LFRIC_LIB on the first two lines.

Reply:
Removed.